main.cpp

```diff
@@ -0,0 +1,147 @@
+#include <iostream>
+#include <string>
+#include <stack>
+#include <vector>
+#include <map>
+
+/*
+ ________  _________  ___  _____  _____   _____
+|  ___|  \/  || ___ \/ _ \/  __ \|  ___| /  __ \ _     _
+| |__ | .  . || |_/ / /_\ \ /  \/| |__   | /  \/| |_ _| |_
+|  __|| |\/| ||    /|  _  | |    |  __|  | |  |_   _|_   _|
+| |___| |  | || |\ \| | | | \__/\| |___  | \__/\|_|   |_|
+\____/\_|  |_/\_| \_\_| |_/\____/\____/   \____/
+
+*/
+
+/**
+ * @brief Get and pop last two items on the stack
+ *
+ * @param stack
+ * @param a
+ * @param b
+ */
+void getTwoTopItems(std::stack<int> &stack, int &a, int &b)
+{
+    a = stack.top();
+    stack.pop();
```

Comment:
could this be condensed down into `a = stack.pop();`?

Reply:
no
as per [ documentaion](https://en.cppreference.com/w/cpp/container/stack/pop) pop returns no value.